OPM/opm-rule-onboarding.ps1

```diff
@@ -498,7 +498,7 @@ Log-Msg -Type Info -MSG "Getting PVWA Credentials to start Onboarding OPM Rules"
 				}
 				
 				if ($objRule.commandGroup -eq $false -and -Not $objRule.command.StartsWith("/") -and -Not $objRule.command -eq ".*") {
-					Write-Warning "Rule Number $counter : It is a recommended security best practice to define commands with an absolute path. Please reconfigure rules like (cat /etc/passwd) to (/usr/bin/cat /etc/passwd)" -WarningAction Inquire
+					Log-Msg -Type Warning -Msg "Rule Number $counter : It is a recommended security best practice to define commands with an absolute path. Please reconfigure rules like (cat /etc/passwd) to (/usr/bin/cat /etc/passwd)" -WarningAction Inquire
```

Comment:
Please note: -WarningAction is not supported by the Log-Msg function and might cause an error

Reply:
I would think it is supported since it takes one of: 		[ValidateSet("Info","Warning","Error","Debug","Verbose")]

Reply:
Oh I see what you are saying